INN/umbrella-citylimits
# Add a Special Projects template, and format its header

## Changes

This pull request makes the following changes:

- adds the series landing page template for special projects
    - supports bylines (not in mockup, but enabled by default in Largo)
    - supports description
    - supports HTML content area (not in mockup, but City Limits has historically used it)
    - supports social media sharing buttons (not in mockup, but enabled by default in Largo)
- does not implement the centered "Special Project" tag from https://user-images.githubusercontent.com/1754187/67224179-a7903780-f3fe-11e9-95d8-2f00e9fa4111.png because that doesn't match how the tag has been styled on the posts widgets in #67's fancy section widgets for the homepage

![Screenshot_2019-11-04 Election Watch 2018 City Limits-mobile](https://user-images.githubusercontent.com/1754187/68155439-b0f9c380-ff17-11e9-9e86-7d502360aa14.png)
![Screenshot_2019-11-04 Election Watch 2018 City Limits-tablet](https://user-images.githubusercontent.com/1754187/68155440-b0f9c380-ff17-11e9-8a1f-e3c8bef6a633.png)
![Screenshot_2019-11-04 Election Watch 2018 City Limits](https://user-images.githubusercontent.com/1754187/68155441-b0f9c380-ff17-11e9-8599-939bbd18ca5c.png)


## Why



For #68 and to give us a template for testing special projects widgets as part of #71

## Testing/Questions

Features that this PR affects:

- no existing features


Questions that need to be answered before merging:

- [ ] should this implement the fancy centered-with-borders styles for the "Special Project" tag?

Steps to test this PR:

1. Create a new series landing page, or edit an existing one.
2. Set the template to Series Landing Page - Special Projects:  
    <img width="294" alt="Screen Shot 2019-11-04 at 15 29 18 " src="https://user-images.githubusercontent.com/1754187/68155533-e69eac80-ff17-11e9-9369-755af549a29f.png">
3. Set the title and description.
4. Set a featured image for the landing page.
5. Save/Update.
6. View the landing page.
